src/components/MenuToolButton.vue

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+import { watch, ref, computed, defineComponent } from 'vue';
+import { useDisplay } from 'vuetify';
+import ToolButton from './ToolButton.vue';
+
+export default defineComponent({
+  name: 'MenuToolButton',
+  props: {
+    icon: { type: String, required: true },
+    name: { type: String, required: true },
+    size: { type: [Number, String], default: 40 },
+    active: Boolean,
+    disabled: Boolean,
+  },
+  components: {
+    ToolButton,
+  },
+  setup(props, { emit }) {
+    const display = useDisplay();
+    const showLeft = computed(() => !display.mobile.value);
+
+    const menuOn = ref(false);
+
+    // Turn off menu if tool clicked and already on
+    const handleActivatorClicked = () => {
+      menuOn.value = !menuOn.value;
+      emit('click');
+    };
+
+    // Turn off menu if tool deselected
+    const active = computed(() => props.active);
+    watch(active, (on) => {
+      if (!on) {
+        menuOn.value = false;
+      }
+    });
+
+    return { showLeft, menuOn, handleActivatorClicked };
+  },
+});
+</script>
+
+<template>
+  <v-menu
+    persistent
```

Comment:
Any reason to make this persistent? I think tool menus don't have a reason to be persistent, especially since we disable close-on-content-click.

Reply:
Persistent keeps the menu open after making an annotation.  Saves a click if you want to make a new annotation with a new label next.

Reply:
While I agree that is useful, I think it goes against users' expectations for how menus operate, and it's not immediately obvious that clicking on the tool again will toggle the menu. The tool menu operates more like a context menu, rather than a persistent menu. Do we have user feedback on having this menu be persistent?

My suggestion would be to add support for controlling tools from the Annotations sidebar, which we can do in a later PR. That would allow users to not have to add an additional click to switch between labels.

Reply:
Makes total sense.  I can work on that next week.

Reply:
I removed the persistent attribute.